airbyte-integrations/bases/debezium-new-version/build.gradle

```diff
@@ -0,0 +1,26 @@
+plugins {
+    id "java-test-fixtures"
+}
+
+project.configurations {
+    testFixturesImplementation.extendsFrom implementation
+}
+dependencies {
+    implementation project(':airbyte-protocol:protocol-models')
+    implementation project(':airbyte-db:db-lib')
+
+    implementation 'io.debezium:debezium-api:1.9.2.Final'
+    implementation 'io.debezium:debezium-embedded:1.9.2.Final'
+//    implementation 'io.debezium:debezium-connector-mysql:1.9.2.Final'
```

Comment:
```suggestion
    // commented out because source-mysql and sqlserver do not yet support the new cdc implementation
//    implementation 'io.debezium:debezium-connector-mysql:1.9.2.Final'
```